<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #rgb(99, 128, 143)</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #435761    ;
border: 0px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: #dfe5e8   ;
background-color: #becbd2   ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>View Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFiles">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: #7d97a5 ;
font-size:14px;
</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
align:center;</string>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>117</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Version Num</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Version Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Added</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>770</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none; background-color: transparent; outline: none;
font: 75 36pt &quot;MS Shell Dlg 2&quot;;
color: #435761;</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:helvetica;
font-size:35px;
color: #435761  ;
</string>
   </property>
   <property name="text">
    <string> File Viewer</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="previewBox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>280</y>
     <width>321</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>250</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryChanges">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Query Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryPast">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Query the Past</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Edit File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="elemHistory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d97a5 ;
  ;  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string> Element Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOld">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Access Past Files</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
